prometheus_es_exporter/nodes_stats_parser.py

```diff
@@ -4,16 +4,25 @@
     'pools': 'pool',
     'collectors': 'collector',
     'buffer_pools': 'buffer_pool',
+    'shards': 'shard'
```

Comment:
It looks like the convention within this file is to have a trailing comma for the last element in the list. If we want to push this to upstream, then I would suggest to stick to the convention.